tests/test_cff.py

```diff
@@ -8,3 +10,76 @@ def test_parse_cff():
         cff_content = f.read()
     authors = get_cff_authors(cff_content)
     assert authors is not None
+
+
+def test_broken_cff():
+    cff_file_emptyish = b"""
+    cff-version: 1.2.0
+    message: "This is a CFF devoid authors or DOI"
+    """
+    cff_file_bad_syntax = b"""
+    cff-version: 1.2.0
+    title: gimie :
+    authors:
+      family-names: Doe
+        given-names: John
+        - family-names: Smith
+    given-names:
+    Jane
+        orcid: 0000-0001-2345-6789
+    """
+    cff_file_broken_orcid = b"""
+    cff-version: 1.2.0
+    title: gimie
+    authors:
+      - family-names: Doe
+        given-names: John
+        orcid: 0000-0001-2345-6789
+      - family-names: Smith
+        given-names: Jane
+        orcid: http://www.orcid.org/0000-0001-2345-6789
+    """
+    cff_file_author_without_orcid = b"""
+    cff-version: 1.2.0
+    title: gimie
+    authors:
+      - family-names: Doe
+        given-names: John
+    """
+
+    # empty graph = \n according to rdflib
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_emptyish)
+        .serialize(format="ttl")
+        == "\n"
+    )
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_bad_syntax)
+        .serialize(format="ttl")
+        == "\n"
+    )
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_broken_orcid)
+        .serialize(format="ttl")
+        == "\n"
+    )
+    assert (
+        CffParser(subject=URIRef("https://example.org/"))
+        .parse(data=cff_file_author_without_orcid)
+        .serialize(format="ttl")
+        == "\n"
+    )
+
+
+def test_doi():
+    cff_file = b"""
+    cff-version: 1.2.0
+    title: gimie
+    doi: 10.5281/zenodo.1234567
+    """
+    assert "https://doi.org/10.5281/zenodo.1234567" in CffParser(
+        subject=URIRef("https://example.org/")
+    ).parse(data=cff_file).serialize(format="ttl")
```

Comment:
**suggestion(non-blocking):** testing if the DOI is among objects would be more robust than in the serialization, also it lets you check if it's indeed a Literal and not URIRef.

Reply:
prob something like

```py
assert Literal("https://...7") in CffParser(...).parse(...).objects()
```